buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/gradlebuild/unittestandcompile/UnitTestAndCompilePlugin.kt

```diff
@@ -186,26 +187,62 @@ class UnitTestAndCompilePlugin : Plugin<Project> {
     }
 
     private
-    fun Project.configureTests() {
-        val javaInstallationForTest = rootProject.availableJavaInstallations.javaInstallationForTest
+    fun Test.getPreviousFailedTestClasses(): Set<String> {
+        val serializer = TestResultSerializer(getBinResultsDir())
```

Comment:
Should use property access: `binResultsDir` instead of `getBinResultsDir()`

Reply:
Done. Thanks.